WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -242,18 +291,46 @@ extension NewOrderViewModel {
     /// Representation of new items in an order.
     ///
     struct NewOrderItem: Equatable, Identifiable {
-        var id: String
-        let product: Product
+        let id: String
+        let productID: Int64
+        let variationID: Int64
```

Comment:
Q: Maybe making this optional can provide better meaning than the "0" value?

Reply:
The `variationID` isn't optional for `OrderItem`, and the API returns `0` for non-variable products, which is why I'm using it that way here. I'll add an explanatory comment about that for clarity.